charts/metering-alm/templates/package.yaml

```diff
@@ -0,0 +1,4 @@
+#! package-manifest: deploy/chart/catalog_resources/rh-operators/metering-operator.v0.12.0.clusterserviceversion.yaml
```

Comment:
This needs to use the `{{ .Values.packageName }}` and `{{ .Values.spec.version }}` values to template in the file name, like the original package-v1.yaml did